modules/ROOT/pages/apikit-workflow-convert-existing.adoc

```diff
@@ -1,104 +1,239 @@
-. In Mappings, click image:add-16x16.png["Add button"] to create a new mapping.
+. In *Mappings*, click `Add` to create a new mapping configuration.
 +
 The *New Mapping* dialog appears.
 . Use the drop-down to map the resources to actions.
 +
-* In the Resource drop-down, select `/users`.
+* In the *Resource* drop-down, select `/users`.
 +
-* In the Action drop-down, select `GET`.
+* In the *Action* drop-down, select `Get`.
 +
-* In the Flow drop-down, select the flow that contains the get users action: `get:\users:api-config`.
+* In the *Flow* drop-down, select the flow that contains the get users action: `GET-Users-FLOW`.
 +
-Click *OK*.
+* Click *OK*.
 +
-image::map-resource-to-action.png["*New Mapping" window with *get:\users:api-config* data populated."]
+. In *Mappings*, click `Add`.
 +
-. Repeat the previous step for each resource-action pairing in the API.
-
-Now, you are ready to run the new project and simulate calls using API Console.
+The *New Mapping* dialog appears.
+. Use the drop-down to map the resources to actions.
++
+* In the *Resource* drop-down, select `/users/userbyid`.
++
+* In the *Action* drop-down, select `Get`.
++
+* In the *Flow* drop-down, select the flow that contains the get users action: `GET-User-By-Id-FLOW`.
++
+* Click *OK*.
+
+[source,xml,linenums]
+----
+<mule>
+    <apikit:config outboundHeadersMapName="outboundHeadersMapName" httpStatusVarName="httpStatus" name="Router" api="api/api.raml" >
+		<apikit:flow-mappings >
+			<apikit:flow-mapping resource="/users" action="get" flow-ref="GET-Users-FLOW" />
+			<apikit:flow-mapping resource="/users/userbyid" action="get" flow-ref="GET-User-By-Id-FLOW" />
+		</apikit:flow-mappings>
+	</apikit:config>
+    <http:listener-config name="HTTP_Listener_config">
+		<http:listener-connection host="0.0.0.0" port="8081" />
+	</http:listener-config>
+	<db:config name="Database_Config">
+		<db:my-sql-connection host="localhost" port="3306" user="admin" password="admin" database="usersdb" />
+	</db:config>
+    <flow name="apiFlow">
+		<http:listener config-ref="HTTP_Listener_config" path="/api/*"/>
+		<apikit:router config-ref="Router"/>
+	</flow>
+	<flow name="GET-Users-FLOW">
+		<db:select config-ref="Database_Config">
+			<db:sql ><![CDATA[SELECT * FROM USERS]]></db:sql>
+		</db:select>
+	</flow>
+	<flow name="GET-User-By-Id-FLOW">
+		<db:select config-ref="Database_Config" >
+			<db:sql ><![CDATA[SELECT * FROM USERS WHERE :userIdParameter == userID]]></db:sql>
+			<db:input-parameters ><![CDATA[#[{userIdParameter: attributes.queryParams.id}]]]></db:input-parameters>
+		</db:select>
+	</flow>
+</mule>
+----
+
+== Scaffolding Approach
+
+Follow the process at xref:apikit-workflow-convert-existing.adoc#step-1-design-your-api-specification[]. When completed, include an alternative workflow for scaffolding your API specification.
```

Comment:
Seems funny not to have this as a step. I think it should be Step 1. Something like this:

Step 1: Follow xref:apikit-workflow-convert-existing.adoc#step-1-design-your-api-specification[] in the previous process.

Reply:
For the lead in to the H2, maybe:

Design your API specification and then include an alternatibe workflow for scaffolding your API specification.

Reply:
I've found a workaround for this structure adding the "Before You Begin" section. I think this new section makes it easier for the reader to understand the steps for each approach.